### ISSUE TITLE
CSLA.NET and Orleans

### ISSUE THREAD

**Issue by tfreitasleal** [MarimerLLC/cslaforum#73]
Microsoft [Orleans project](http://dotnet.github.io/orleans) looks very interesting. I can't remember where I read first about it and I thought maybe it was here. I couldn't find any reference to Orleans on Csla .NET repositories.

Is anyone using Csla .NET and Orleans? Any findings to share?

**Comment by rockfordlhotka** [comment]
I agree, Orleans is some cool stuff. Personally I haven't had time to dig into it though :disappointed:
